discovery-aws-api/src/main/scala/akka/discovery/awsapi/ecs/EcsSimpleServiceDiscovery.scala

```diff
@@ -22,9 +22,11 @@ import scala.concurrent.duration._
 
 class EcsSimpleServiceDiscovery(system: ActorSystem) extends SimpleServiceDiscovery {
 
-  private[this] implicit val ec: ExecutionContext = system.dispatcher
+  private[this] val cluster = system.settings.config.getString("akka.discovery.aws-api-ecs.cluster")
```

Comment:
please first `val c = system.settings.config.getConfig("akka.discovery.aws-api-ecs")` and then get values from that one

Reply:
I would definitely have written it as `val config = ...` if two or more values were being read from it. It just seemed a bit redundant as long as there's only one thing being read. I take it that's the preferred style in either case though, so I've made and pushed the change.